adsdata/app.py

```diff
@@ -5,32 +5,62 @@
 import os
 from sqlalchemy.orm import sessionmaker
 
-import columnFileIngest
 import row_view
 import metrics
+import reader
 from adsputils import load_config, setup_logging
 
 logger = None
 config = {}
 
+def test_load_column_files(sql_sync):
+    """ use cursor/psycopg.copy_from to data from column file to postgres
+    
+    after data has been loaded, join to create a unified row view 
+    """
 
-def load_column_files(sql_sync):
-    """ generate and run sql 'copy from program' commands to load column files
-    and, create joined row view 
-    note that the copy from program command actually runs this python code with different arguments
+    conn = sql_sync.engine.raw_connection()
+    cur = conn.cursor()
+    table_name = sql_sync.schema + '.' + 'canonical'
+    f = reader.BibcodeFileReader('/Users/SpacemanSteve/tmp/bibcodes.list.can.20170522') #../tests/data/testBibcodes120.txt')
```

Comment:
yep, it was temporary and has been deleted.  code now in tests/tests/test_rowview_ingest.py